src/test/kotlin/domain/racing/NameTest.kt

```diff
@@ -0,0 +1,23 @@
+package domain.racing
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class NameTest {
+    @ParameterizedTest
+    @ValueSource(strings = ["", "  "])
+    fun `자동차 이름은 빈값일 수 없다`(name: String) {
+        Assertions.assertThatThrownBy { Name(name) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+            .hasMessageContaining("자동차 이름은 빈값일 수 없습니다.")
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = ["123456", "1234567"])
+    fun `자동차 이름은 5자를 초과할 수 없다`(name: String) {
+        Assertions.assertThatThrownBy { Name(name) }
+            .isInstanceOf(IllegalArgumentException::class.java)
+            .hasMessageContaining("자동차 이름은 5자를 초과할 수 없습니다.")
```

Comment:
assertThrows를 사용하면 Reflection을 사용하지 않아도 됩니다!
```suggestion
        assertThrows<IllegalArgumentException>("자동차 이름은 5자를 초과할 수 없습니다.") { Name(name) }
```

Reply:
더 간결하네요 👍